charts/tfy-gpu-operator/values.yaml

```diff
@@ -523,91 +525,9 @@ civo-talos-nvidia-device-plugin:
   ## @skip civo-talos-nvidia-device-plugin.tolerations
   ##
   tolerations:
-    - operator: Exists
-
-
-civo-talos-dcgm-exporter:
```

Comment:
This is okay to remove because we have removed dcgm component in last release. We will bring it back using the gpu operator route later